qa/full-cluster-restart/src/test/java/org/elasticsearch/upgrades/FullClusterRestartIT.java

```diff
@@ -119,6 +120,10 @@ public void testSearch() throws Exception {
             client().performRequest("POST", "/" + index + "/_refresh");
         }
         assertBasicSearchWorks(index);
+        assertAllSearchWorks(index);
+        assertBasicAggregationWorks(index);
+        assertRealtimeGetWorks(index);
+        assertNewReplicasWork(index);
```

Comment:
I feel like `assertNewReplicasWork` should be in its own test. It doesn't have much to do with searching, I don't think.